src/locales/da/admin.ftl

```diff
@@ -355,6 +355,7 @@ moderate-marker-link = Link
 moderate-marker-bannedWord = Forbudt ord
 moderate-marker-suspectWord = Mistænkte ord
 moderate-marker-spam = Spam
```

Comment:
Old keys should be removed.

Reply:
It's still being used for the `COMMENT_REPORTED_SPAM` marker flag. If we remove it, that functionality will no longer work.